src/Routes.js

```diff
@@ -196,36 +116,39 @@ export const Routes = () => {
                 </Bullseye>
             }
         >
-            {!hasSystems ? (
-                <AsyncComponent
-                    appId="content_management_zero_state"
-                    appName="dashboard"
-                    module="./AppZeroState"
-                    scope="dashboard"
-                    ErrorComponent={<div>Error state</div>}
-                    app="Content_management"
-                />)
-                :
-                <Switch>
-                    {paths.map(PermissionRouter)}
-                    <Redirect
-                        from='/advisories/:advisoryId/:inventoryId'
-                        to='/systems/:inventoryId'
-                    />
-                    <Redirect
-                        from='/packages/:packageName/:inventoryId'
-                        to='/systems/:inventoryId'
-                    />
-                    <Route render={() =>
-                        (
-                            (!some(paths, p => p.to === history.location.pathname)) && (
-                                <Redirect to={'/advisories'} />
-                            )
-                        )
-                    }
-                    />
-                </Switch>
-            }
+            <Router>
```

Comment:
Something is not right here. This should throw an error like "Router in Router not allowed".

Reply:
🙈 my bad. I didn't see the `as` for the import. But maybe we can avoid that renaming. What if we use `AdvisorRoutes` for this const and not rename the `Routes` component from react router?